common/data/data.json

```diff
@@ -1117,6 +1141,12 @@
     "seaweed": "<:seaweed:1043637964158279742>",
     "mask": ":mask:",
     "cornucopia": "<:cornucopia:1041714283924422836>",
+    "rudolfs_red_nose": "<:rudolfs_red_nose:1047335258598670356>",
+    "tree": "<a:traditional_christmas_tree:1047335222708011058>",
+    "mistletoe": "<:mistletoe:1047336566638526545>",
+    "inflate_santa": "<:inflate_santa:1047337111151464458>",
+    "tree_globe": "<:tree_globe:1047337839068729364>",
+     "s_sack": "<:s_sack:1047338463097266217>",
```

Comment:
There's a weird space here and I would prefer this be named `santa_sack` or `santas_sack`